practice_app/apps/user/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+class User(models.Model):
+    username = models.CharField(max_length=50, unique = True)
+    password = models.CharField(max_length=50)
+    mail = models.CharField(max_length=50)
```

Comment:
Similar to password you can make this field as email type. You can use **models.EmailField**, to get a better understanding have a look at [this link](https://www.geeksforgeeks.org/emailfield-django-models/).

Reply:
Thanks again for the link, I wil also work on that.